docs/_articles/en/v3.x/salesforce-graph-engine/rules.md

```diff
@@ -19,7 +19,8 @@ ApexFlsViolationRule detects [Create, Read, Update, and Delete (CRUD) and Field-
 |				 |`public`-scoped methods on Visualforce Controllers																|
 |				 |`global`-scoped methods on any class																			  	|
 |				 |`Messaging.InboundEmailResult handleInboundEmail()` methods on implementations of `Messaging.InboundEmailHandler`	|
-|				 |Any method targeted during invocation																|
+|				 |Any method targeted during invocation			
+|				 |Entry points, not including triggers
```

Comment:
This feels weird, because all of the above items are also entry points. Is there a better way for us to say that triggers aren't currently supported?